docs/docs/1.2.0/documentation/dps-and-data-model.md

```diff
@@ -341,22 +341,30 @@ If we want to expose a `Service` in one zone only (as opposed to multi-zone), we
 
 For an in-depth example on deploying Kuma with [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller), please follow this [demo application guide](https://github.com/kumahq/kuma-demo/tree/master/kubernetes).
 
-## Ingress
+## Zone Ingress
 
-To implement cross-zone communication when Kuma is deployed in a [multi-zone](/docs/1.1.6/documentation/deployments/#multi-zone-mode) mode, the `Dataplane` model introduces the `Ingress` mode. These data plan proxies are not attached to any particular workload. Instead, they are bound to that particular zone.
-All requests that are sent from one zone to another will be directed to the proper instance by the Ingress.
-The specifics of the `Ingress` data plane proxy are described in the `networking.ingress` dictionary in the YAML resource.
-Ingress has a regular address and one inbound just like a regular data plane proxy. This address is routable within the local Ingress zone. It also has the following public coordinates:
-* `networking.ingress.publicAddress` - an IP address or hostname which will be used by data plane proxies from other zones
-* `networking.ingress.publicPort` - a port which will be used by data plane proxies from other zones
+To implement cross-zone communication when Kuma is deployed in a [multi-zone](/docs/1.1.6/documentation/deployments/#multi-zone-mode) mode, there is a new proxy type `Zone Ingress`. These proxies are not attached to any particular workload. Instead, they are bound to that particular zone.
+All requests that are sent from one zone to another will be directed to the proper instance by the `Zone Ingress`.
 
-Ingress that don't have this information is not taken into account when generating Envoy configuration, because they cannot be accessed by data plane proxies from other zones.
+The `Zone Ingress` entity includes a few sections:
+
+* `type`: must be `ZoneIngress`.
+* `name`: this is the name of the zone-ingress instance, and it must be **unique** for any given `Zone`.
```

Comment:
is the `zone` object lower-case ("z") or upper-case ("Z"). Not completely consistent here.